mods/cnc/rules/defaults.yaml

```diff
@@ -813,6 +813,26 @@
 	RenderSprites:
 		Palette: terrain
 
+^CivHaystack:
+	Inherits: ^CivField
+	MapEditorData:
+		ExcludeTilesets: DESERT
+	-Tooltip:
+	GrantConditionOnTileSet@WINTER:
+		Condition: winter
+		TileSets: SNOW, WINTER
+	Tooltip@WINTER:
+		Name: Igloo
+		RequiresCondition: winter
+		GenericVisibility: Enemy, Neutral
+		ShowOwnerRow: false
+	GrantConditionOnTileSet@SUMMER:
+		Condition: summer
+		TileSets: TEMPERAT
+	Tooltip@SUMMER:
+		Name: Haystacks
+		RequiresCondition: summer
```

Comment:
`RequiresCondition: !winter` and `GrantConditionOnTileSet@SUMMER` can be removed.